client/jetpack-onboarding/summary-completed-steps.jsx

```diff
@@ -23,23 +23,31 @@ import {
 	JETPACK_ONBOARDING_STEPS as STEPS,
 } from './constants';
 
-const CompletedSteps = ( { siteSlug, steps, stepsCompleted, stepsPending } ) =>
+const CompletedSteps = ( {
+	handleNextStepSelection,
```

Comment:
Do we want to have a default `noop` value for `handleNextStepSelection`? Right now if you don't pass a `handleNextStepSelection` prop to `CompletedSteps`, it'll blow up.